### ISSUE TITLE
"Articulation" of Ribs - need way to associate seriated ribs

### ISSUE THREAD

**Issue by tlvandeest** [spawaskar-cora/cora-docs#493]
There is currently no way to record an association between seriated ribs from one side (e.g.,  Right Rib 1, Rib 2, and Rib 3 all belong together).  Even when multiple ribs are entered as a Rib Group, when you go into each individual rib there is no indication it is associated with the adjacent ribs.  Other groups (hand, foot, vertebrae) automatically create articulations.  While ribs do not truly articulate with one another, this may be the easiest way to capture their association within the thorax (as opposed to building something within CoRA specifically for this rib situation).  @fedamann @SachinPawaskarUNO @clegarde

**Comment by clegarde** [comment]
Good point! What if we added another association like “Morphology” that could suit multiple scenarios? Thinking about a recent case where two metacarpals were associated together based on their morphology/size and not articulation.

**Comment by tlvandeest** [comment]
Talked with @emilystreetman about this also and we concluded that a Morphology category for association would be a good option but were not clear on how often that would be useful against the effort/time/need to build another layer into the system.  I think this would be a good discussion point for the next scrum.

**Comment by SachinPawaskarUNO** [comment]
related to #511

**Comment by CTegtmeyer** [comment]
Marking this issue as resolved now that the morphology option has been added into CoRA.
